components/dataViews/MultisigHoldings.tsx

```diff
@@ -15,8 +13,10 @@ const MultisigHoldings = (props: Props) => {
     <StackableContainer lessPadding fullHeight>
       <h2>Holdings</h2>
       <StackableContainer lessPadding lessMargin>
-        {props.holdings ? (
-          <span>{printableCoin(props.holdings, state.chain)}</span>
+        {props.holdings.length ? (
+          props.holdings.map((holding) => (
+            <span key={holding.denom}>{printableCoin(holding, state.chain)}</span>
+          ))
```

Comment:
This is a list of spans. How is that shown? Would be good to have a list.

You can try https://www.mintscan.io/juno/account/juno190g5j8aszqhvtg7cprmev8xcxs6csra7xnk3n3 with juno mainnet, which has 3 coins

Reply:
It looks like this because 2 of those are IBC:

![image](https://user-images.githubusercontent.com/44572727/231768431-aeff894d-7fb5-4c3e-a3f8-a42e95f17045.png)

Reply:
I can fix the overflow style and maybe put each coin into its own box, not sure about the denom

Reply:
We can just use [ellideMiddle](https://github.com/CosmWasm/code-explorer/blob/main/src/ui-utils/index.ts#L4-L12) for long denoms. What's inportant to show there is the "ibc/" part in the beginning and a few characters in the even. So we can ellide to 12 chars or something.